src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/service/JJDisputeService.java

```diff
@@ -59,7 +115,7 @@ public List<JJDispute> getAllJJDisputes(String jjAssignedTo) {
 	 * @return
 	 */
 	@Transactional
-	public JJDispute updateJJDispute(String id, JJDispute jjDispute, CustomUserDetails user) {
+	public JJDispute updateJJDispute(String id, JJDispute jjDispute, Principal principal) {
```

Comment:
Just to confirm that you want to remove CustomUserDetails in favour of Principal.  It think Burak just did the opposite and just added the CustomUserDetails .

Reply:
When generating the oracle data C# api code it continuously generated the 'user' variable in the body of the request and so was incompatible with passing the information in the headers.  When using the 'principal' variable, it did not attempt to generate it in the body of the request. I think it is a problem we need to examine for future reference, I did prefer his approach.